coala_quickstart/Strings.py

```diff
@@ -50,3 +50,18 @@
 
 You can see all of them here: {}
 """.format(BEAR_DOCS_URL)
+
+PRINT_BEARS = {'unusable':
+               {'msg': """
+Based on the configuration options the following bears have been identified
+as unusable (options '-C' or '--ci' (non interactive mode) make the bears that
```

Comment:
language trouble, parenthesis shouldn't be here starting before the options

Reply:
imo, the sentence is too long too, needs to split.

Reply:
currently, there's nesting of braces, and the sentence inside the braces is long too.

Reply:
How about this: "The following bears have been dropped because '-C'/'--ci' (non interactive mode) option has ben selected. Remove the option or check '--allow-incomplete-sections' option for more information on how to include them:" ?

Reply:
sounds good.